src/guides/v2.4/release-notes/commerce-2-4-1.md

```diff
@@ -683,27 +858,123 @@ We have fixed hundreds of issues in the Magento 2.4.1 core code.
 
 <!--- MC-32956-->
 
-*  The position of products in the `catalog_category_product` table now update as expected when an administrator creates a product in the Admin and assigns it to a category. Previously, the position of new products was always assigned a 0 value.
+*  The position of products in the `catalog_category_product` table now updates as expected when an administrator creates a product in the Admin and assigns it to a category. Previously, the position of new products was always assigned a 0 value.
 
 <!--- MC-34939-->
 
 *  Customer data is now successfully exported from the Admin, and the export data grid displays customer data as expected. Previously, an error related to memory allocation occurred during export.
 
 <!--- MC-34657-->
 
-*  Imported `.cvs` files now capture related product information as expected. Previously, related product information was not consistently uploaded the first time the `.cvs` file was imported.
+*  Imported CVS files now capture related product information as expected. Previously, related product information was not consistently uploaded the first time the CVS file was imported.
 
-### Index
+<!--- MC-37424-->
 
-<!--- ENGCOM-7776-->
+*  Removed unused construct parameters in `AdvancedPricing.php`. [GitHub-29531](https://github.com/magento/magento2/issues/29531)
+
+### Index
 
 <!--- ENGCOM-7073-->
 
-<!--- MC-39468-->
+*  `Magento_CacheInvalidate` now handles large tag patterns correctly when doing a `PURGE`. `sendPurgeRequest` has been refactored to handle an array of tags instead of requiring the caller to use `implode()`.
 
 <!--- MC-30568-->
 
-*  Shared indexers now show a status of **valid** after you run `bin/magento indexer:status` after re-indexing. Previously, shared indexers had an **invalid** status after a full re-index.
+*  Shared indexers now show a status of **valid** when you run `bin/magento indexer:status` after re-indexing. Previously, shared indexers had an **invalid** status after a full re-index.
```

Comment:
This fix was already included in 2.4.0, so this should get moved to the 2.4.0 release notes where it seems to be missing.

Reply:
I've removed it from these notes and will add it to the 2.4.0 notes in a separate PR. I'm glad you caught this -- the fix version was incorrect in JIRA, which is why my query picked it up.